File: Algorithms/p075_Sort_Colors/p075_Sort_Colors_1.py
class Solution(object):
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        cnt = [0]*3
        for v in nums:
            cnt[v] += 1
        i = 0
        for j in range(0,3):
            while cnt[j]>0:
                nums[i] = j
                i += 1
                cnt[j] -= 1

File: Algorithms/p075_Sort_Colors/p075_Sort_Colors_1.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        vector<int> cnt(3,0);
        for(int v : nums){
            ++cnt[v];
        }
        for(int i=0,j=0;j<3;++j){
            for(;cnt[j];--cnt[j]){
                nums[i++] = j;
            }
        }
    }
};
